numpy/f2py/crackfortran.py

```diff
@@ -1522,10 +1522,10 @@ def markinnerspaces(line):
             continue
         if f == 0 and c in ['\'', '"']:
             cc = c
-        if c == cc:
```

Comment:
Now that you are working on this, it would be useful to add a one-sentence or so docstring to the function, that says what it is intended to do.

Reply:
done